virt/backup_restore/virt-backup-restore-overview.adoc

```diff
@@ -3,45 +3,52 @@ include::_attributes/common-attributes.adoc[]
 [id="virt-backup-restore-overview"]
 = Backing up and restoring virtual machines
 :context: virt-backup-restore-overview
+:virt-backup-restore-overview:
+:credentials: cloud-credentials
+:provider: gcp
```

Comment:
Are these temporary attributes unset somewhere that I have missed?

(also related to the "Conditionals for different assemblies" section of the contribution guide) https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#conditionals-for-different-assemblies